### ISSUE TITLE
The pre-trained TransVW and the provided self-discoverd 3D visual words

### ISSUE THREAD

**Issue by chengaox** [fhaghighi/TransVW#1]
Hello,
I can NOT find where to download the pre-trained TransVW (TransVW_chest_ct.pt) and the provided self-discoverd 3D visual words as well as their pseudo labels?
Thanks

**Comment by fhaghighi** [comment]
Hi, 

Thanks for your interest in our work. The link to the self-discovered data has been updated; so, you should be able to download it now from the Keras and Pytorch pages (also here is the [link](https://zenodo.org/record/4625321/files/TransVW_data.zip?download=1) for your convenience). The pre-trained model in Keras is already available; Pytorch model will be released soon as well.

**Comment by fhaghighi** [comment]
Hi, please use the updated link to download our Pytorch model. Also, here is the download [link](https://zenodo.org/record/4625321/files/TransVW_chest_ct.pt?download=1) for your convenience.
